### DISCUSSION TITLE
Please help me to understand why for some tickers the library is not reflecting yahoo official data for closure

### DISCUSSION THREAD

**Discussion by MarcoOvidi** [ranaroussi/yfinance#2163] [Yahoo's data]
### Describe bug

For example for ticker 1AKAM.MI the closure from yahoo website is 89.35 at 27/11/2024, but the library is reporting a closure of 80.00 for the same date. Am I filling wrong the request to the library ? 

https://finance.yahoo.com/quote/1AKAM.MI/

### Simple code that reproduces your problem

df = yfinance.Ticker(('1AKAM.MI').history(start=start_date, auto_adjust=False) o 
df = yfinance.Ticker('1AKAM.MI').history(start=start_date)

### Debug log

not needed I think

### Bad data proof

<img width="469" alt="Screenshot 2024-11-30 alle 17 08 57" src="https://github.com/user-attachments/assets/6489a7ba-4d4a-4b0f-8f24-28088582fc88">

### `yfinance` version

yfinance==0.2.50

### Python version

python                    3.13.0

### Operating system

Mac Sequoia 15.1

**Comment by dhruvan2006** [comment]
> the closure from yahoo website is 89.35 at 27/11/2024

Where are you getting this information from?

There doesn't seem to be anything wrong for me
![image](https://github.com/user-attachments/assets/00333701-1e6a-4d5e-9109-c5fbb1ee03b5)

**Reply by MarcoOvidi** [reply]
ok , my mistake, yahoo finance is reflecting yfinance, others sources were different, thank you in any case :-)
